src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -772,4 +773,24 @@ BOOST_AUTO_TEST_CASE(HardShrinkFunctionTest)
                                    desiredDerivatives);
 }
 
+/**
+ * Basic test of the Elliot function.
+ */
+BOOST_AUTO_TEST_CASE(ElliotFunctionTest)
+{
+  // calculated using pytorch tensor
```

Comment:
Hi, Could also capitalize the first letter of the comment and add a stop at the end. Thanks.

Reply:
Thanks for pointing out, changing...